metro.config.js

```diff
@@ -31,6 +31,50 @@ if (process.env.CI) {
 const rainbowConfig = {
   resolver: {
     blacklistRE,
+    resolveRequest: (context, moduleName, platform) => {
```

Comment:
This is required to resolve ledger packages correctly otherwise it won't find them.